Recipe-App/Recipe-App/UIViews/RecipeListView.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  RecipeListView.swift
+//  Recipe-App
+//
+//  Created by Sphelele Ngubane on 2021/06/11.
+//
+
+import SwiftUI
+
+struct RecipeListView: View {
+    
+    @State var searchText: String = ""
```

Comment:
shouldn't this be a binding property ?

Reply:
We can use the binding on another view that may want to use the searchText.